lambda/custom/document/renderPage.json

```diff
@@ -0,0 +1,138 @@
+{
+    "type": "APL",
+    "version": "1.0",
+    "settings": {},
+    "theme": "dark",
+    "import": [
+            { 
+                "name" : "alexa-styles", 
+                "version" : "1.0.0" 
+            } 
+        ],
+    "resources": [
+        { 
+            "colors" : {
+               "SkillBackgroundColor" : "blue" , 
+               "SkillHighlightTextColor"  : "yellow" , 
+               "SkillTextColor" : "white"
+            }
+        } 
+        
+        ],
+    "styles": {
+        
+        
+    },
+    "onMount": [],
+    "graphics": {},
+    "commands": {},
+    "layouts": {
+        "CenteredText" : 
+        { 
+            "parameters" :  [
+                "text", 
+                "speech", 
+                "cid" 
+                ] , 
+            "item" : 
+            {
+                "id" : "${cid}" , 
+                "type" : "Text", 
+                "text" : "${text}", 
+                "speech" : "${speech}", 
+                "textAlign" : "center" , 
+                "color" :  "@SkillTextColor" , 
+                "width" : "50%", 
+                "height" : "100%" , 
+                "textAlignVertical": "center" ,  
+                "fontSize" : "@fontSizeRegular" 
+            }
+        }, 
+        "SkillLogo" : 
+        { 
+            "type" : "Frame" , 
+            "item" : { 
+                "type" : "Image" , 
+                "source" : "" 
```

Comment:
This object needs width and height defined in APL 1.1

Reply:
will do